bin/pre_download_compilers

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# This ./cache/hardhat-nodejs is coming from the env-paths module
+# that hardhat is using.
+COMPILER_DIR=/root/.cache/hardhat-nodejs/compilers-v2
+mkdir -p $COMPILER_DIR/linux-amd64
```

Comment:
you could make mkdir -p $COMPILER_DIR/{/linux-amd64,/vyper/linux,/zksolc,/zkvyper}

Reply:
nice, corrected